app/settings.ts

```diff
@@ -150,6 +150,7 @@ export class LocalSettings extends Settings<LocalSettingsData> {
   private async createDefaults(): Promise<LocalSettingsData> {
     return {
       version: VERSION,
+      openOnStart: false,
```

Comment:
This is not enough to do when adding new settings. This only creates the default values if the settings file doesn't exist, i.e. when users first install the application. Additionally, we need to add new settings to the existing setting files for people who already have the application installed.

You can easily do this by increasing the `VERSION` (for local settings) number in this file and then adding another block in the `migrateOldSettings` function which checks for that version number and adds the new setting.